### PR TITLE
fix(deps): update to kuromojin@3

### PR DESCRIPTION
**Pull request by bizen241**
## Summary

Update to kuromojin@3 and improve analysis.

## Fixes

- update to kuromojin@3
  - refs azu/kuromojin#8
- update to lint-staged@10.5.4
- update to prettier@2.2.1
- use core.hooksPath instead of husky

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,38 +25,33 @@
...
   "keywords": [
     "textlint",
     "textlintrule"
   ],
   "description": "ら抜き言葉を検出するtextlint rule",
   "devDependencies": {
-    "husky": "^4.3.0",
-    "lint-staged": "^10.4.2",
+    "lint-staged": "^10.5.4",
     "power-assert": "^1.4.1",
```
Comment (azu) [inline package.json]: Probably, we can remove `power-assert` dependencies.
It looks no longer used.

**Review by azu — COMMENTED** [review]
Thanks!

File: package.json (hunk 1 above)
Comment (bizen241) [reply package.json]: Thank you for reviewing.
I'll remove it.

**Review by azu — APPROVED** [review]
LGTM!

**Comment by azu** [comment]
Release https://github.com/textlint-ja/textlint-rule-no-dropping-the-ra/releases/tag/v3.0.0 🎉

**Pull request merged**